src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
```

Comment:
why do we need this ?

Reply:
This is under the folder structure of create-react-app package. We can delete or rename these  files. I'm planning to remove and test locally and update it for the next version.

Reply:
thanks for clarifying